sdk/search/Azure.Search/src/autorest.md

```diff
@@ -178,9 +125,25 @@ directive:
   transform: $.type = "object";
 ```
 
-### Add default responses
-TODO...
+### Make Loookup Document behave a little friendlier
+It's currently an empty object and adding Additional Properties will generate
+a more useful model.
+``` yaml
+directive:
+- from: swagger-document
+  where: $.paths["/docs('{key}')"].get.responses["200"].schema
+  transform:  >
+    $.additionalProperties = true;
+```
 
 ## C# Customizations
 Shape the swagger APIs to produce the best C# API possible.  We can consider
 fixing these in the swagger files if they would benefit other languages.
+
+### QueryType
+``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions.QueryType['x-ms-enum']
+  transform: $.name = "SearchQueryType";
```

Comment:
Did you rename here because you didn't want to copy all the values? [CodeGenSchema] should be supported on enums now.

Reply:
No - I just couldn't get it working with `CodeGenSchema` when I tried.  Let me check again.

Reply:
Changed